exampleSite/config.toml

```diff
@@ -2,6 +2,7 @@ baseURL = "https://example.com"
 languageCode = "en-us"
 title = "Some Title"
 theme = "minimal-bootstrap-hugo-theme"
+themesDir = "../.."
```

Comment:
This needs to be removed, else it will trip up regular users of this theme. 
If you need this for Stackbit, I would add it to your build command instead.

Reply:
removed in the commit 6388023